### PR TITLE
feat: Add favorite button

### PR DESCRIPTION
**Pull request by Merkur39**

### CODE REVIEW

File: src/connections/allContacts.js [hunk 1]
```diff
@@ -38,6 +38,11 @@ export const importContact = async (client, attributes) => {
   return client.create(DOCTYPE_CONTACTS, attributes, null)
 }
 
+export const updateContact = ({ client, contact, attributes }) => {
+  const updatedContact = merge({}, contact, attributes)
+  return client.save(updatedContact)
+}
```
Comment (zatteo) [inline src/connections/allContacts.js:44]: nit: 

I saw this in the same file, maybe it does the same thing ? 
```
const updatedContact = mergeContact(contacts[0], attributes)
    return client.save(updatedContact)
```

**Comment by zatteo** [comment]
I think it misses the ability to favorite/unfavorite in the selection bar ?
image

File: src/connections/allContacts.js (hunk 1 above)
Comment (Merkur39) [reply src/connections/allContacts.js:44]: Bien vu, en effet toute une logique a été faite autour de ça, autant l'utiliser 👍

**Comment by Merkur39** [comment]
> I think it misses the ability to favorite/unfavorite in the selection bar ?

That's right, it's added. And I've modified a few previous commits

**Review by zatteo — APPROVED** [review]

**Pull request merged**